src/libs/Navigation/setupCustomAndroidBackHandler/index.android.ts

```diff
@@ -22,12 +22,6 @@ function setupCustomAndroidBackHandler() {
             return false;
         }
 
-        const isLastScreenOnStack = bottomTabRoutes.length === 1 && rootState?.routes?.length === 1;
-
-        if (NativeModules.HybridAppModule && isLastScreenOnStack) {
-            NativeModules.HybridAppModule.exitApp();
-        }
-
```

Comment:
Why has this code been deleted?

Reply:
When ND was on home screen and back button was clicked app crashed instead of going to background